plugins/extra/p2js/components/P2BodyMarker.ts

```diff
@@ -52,6 +54,12 @@ export default class P2BodyMarker extends SupEngine.ActorComponent {
     this.mesh.updateMatrixWorld(false);
   }
 
+  setAngle(angle: number) {
+    this.angle = angle;
```

Comment:
I would rather do the conversion from degrees to radians here so we don't have to worry about it elsewhere.